static/js/controllers/contacts-content.js

```diff
@@ -264,6 +264,24 @@ var _ = require('underscore'),
         });
       };
 
+      var addAdditionalDetails = function(reports, children) {
```

Comment:
Sure. FWIW the naming was to indicate that if we were going to do anything more like that in the future (e.g. there is a place name we're not filling in correctly either, though I don't think it's ever there for JSON reports) it could go here as well.